### PR TITLE
Symphony producers

### PR DESCRIPTION
**Pull request by jgreben**

### CODE REVIEW

File: Demos/ReactiveKafkaFsProducer/src/main/scala/ReactiveKafkaFsProducer.scala [hunk 1]
```diff
@@ -25,19 +26,22 @@ object ReactiveKafkaFsProducer extends App {
...
+
   val path                  = File(s"${dir}/Casalini_mrc").path.toString
 
   val bootstrapServers = config.getOrElse("bootstrapServers", "").toOption.fold("")(identity(_))
 
   val producerSettings = ProducerSettings(system, new StringSerializer, new ByteArraySerializer)
     .withBootstrapServers(bootstrapServers)
 
-  val source: Source[Path, NotUsed] = Directory.ls(FileSystems.getDefault.getPath(path))
+  val path                          = inDir.path.toString
```
Comment (atz) [inline Demos/ReactiveKafkaFsProducer/src/main/scala/ReactiveKafkaFsProducer.scala]: This second definition of `path` is causing syntax errors

**Review by atz — DISMISSED** [review]

File: Demos/ReactiveKafkaFsProducer/src/main/scala/ReactiveKafkaFsProducer.scala (hunk 1 above)
Comment (jgreben) [reply Demos/ReactiveKafkaFsProducer/src/main/scala/ReactiveKafkaFsProducer.scala]: Thanks, got it now.

File: Demos/ReactiveKafkaFsProducer/src/main/scala/ReactiveKafkaFsProducer.scala [hunk 2]
```diff
@@ -25,19 +26,21 @@ object ReactiveKafkaFsProducer extends App {
...
+
   val path                  = File(s"${dir}/Casalini_mrc").path.toString
 
   val bootstrapServers = config.getOrElse("bootstrapServers", "").toOption.fold("")(identity(_))
 
   val producerSettings = ProducerSettings(system, new StringSerializer, new ByteArraySerializer)
     .withBootstrapServers(bootstrapServers)
 
-  val source: Source[Path, NotUsed] = Directory.ls(FileSystems.getDefault.getPath(path))
+  val fs                            = FileSystems.getDefault
```
Comment (atz) [inline Demos/ReactiveKafkaFsProducer/src/main/scala/ReactiveKafkaFsProducer.scala]: AFAICT, the changes to this file are not semantic.  They are just backing out some changes I made in a previous commit, so I don't think you intend them to be part of this PR.  Can you exclude them?

**Review by atz — DISMISSED** [review]

**Comment by atz** [comment]
@jgreben my point was that you don't seem to *intend* any changes to that file at all.  The diff seems to be an artifact of your work starting from an earlier position in the git history (i.e. not being rebased).  Maybe we can pair to rebase and squash these changes into one commit touching two files?

**Pull request closed without merging**